atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -68,14 +78,27 @@
 
     private List<InetSocketAddress> cassandraHosts;
 
+    private volatile Set<InetSocketAddress> localHosts;
+    private final Supplier<Optional<HostLocation>> myLocationSupplier;
+
+    private final Counter randomHostsSelected;
+    private final Counter localHostsSelected;
+
     public CassandraService(MetricsManager metricsManager, CassandraKeyValueServiceConfig config, Blacklist blacklist) {
         this.metricsManager = metricsManager;
+        this.randomHostsSelected = metricsManager.getTaggedRegistry().counter(
+                MetricName.builder().safeName("randomHostsSelected").build());
+        this.localHostsSelected = metricsManager.getTaggedRegistry().counter(
+                MetricName.builder().safeName("localHostsSelected").build());
```

Comment:
ah - yes, noticed you missed something. It's important that we namespace our metrics, since otherwise in metrics systems it's unclear who's producing what. We typically do this by writing something like `MetricRegistry.name(CassandraService.class, "localHostsSelected)`